src/adapter/properties.rs

```diff
@@ -465,3 +465,58 @@ pub(crate) fn resolve_associated_constant_property<'a>(
         _ => unreachable!("AssociatedConstant property {property_name}"),
     }
 }
+
+// const example from rustdocs
+// {"id"           	:"0:9:xx","crate_id":0,
+// "name"          	:"PUB_CONST_NAME",
+// "span"          	:{"filename":"","begin":[5,0],"end":[5,68]},
+// "visibility"    	:"public",
+// "docs"          	:null,
+// "links"         	:{},
+// "attrs"         	:[],
+// "deprecation"   	: null,
+// "inner"         	: { ← &inner will be set to ItemEnum::Constant(c), where c=struct rustdoc_types::Constant
+//   "constant"    	: { rustdoc_types::Constant has type_/expr/value/is_literal fields (docs.rs/rustdoc-types/latest/rustdoc_types/struct.Constant.html)
+//     "type"      	: currently not supported as a property pending custom scalar type support in Trustfall (https://github.com/obi1kenobi/trustfall-rustdoc-adapter/pull/280#discussion_r1348950734)
+//     "type"      	:{"primitive":"i32"}, or ↓ or any of docs.rs/rustdoc-types/latest/rustdoc_types/enum.Type.html
+//     "type"      	:{"borrowed_ref":{"lifetime": "'static","mutable": false,"type":{"primitive": "str"}}},
+//     "type"      	:{"resolved_path":{"name":"Years","id": "0:3:1633","args":{"angle_bracketed":{"args":[],"bindings":[]}}}},
+//     "expr"      	: "_",
+//     "value"     	: null,
+//     "is_literal"	: false}}
+// },
```

Comment:
Would you mind if we removed this block? While it's nice to see what the underlying representation is, the JSON format is quite unstable (it changes on average once per Rust version) and I have a strong feeling that we'll forget to keep this updated to match the current format. The only thing worse than no examples is wrong examples, and I think this will very quickly become a wrong example.

Also, even on day 1 it will require lots of extra manual work to recreate this example in all the other `trustfall-rustdoc-adapter` versions I maintain, and I won't be able to use my usual PR-porting script to do it.

```suggestion
```